### PR TITLE
Enable testing 4K in the test page (issue 963)

### PR DESCRIPTION
**Pull request by rantonysamy**
**Description**
Adding 4k button in the testing page to test 4k camera and its resolution. 

**Purpose**
4096x2160 (4K) has to be added in the page https://webrtc.github.io/samples/src/content/getusermedia/resolution/ 

Added button 4k and all the related code to bring this button to test the resolution.

### CODE REVIEW

File: src/content/getusermedia/resolution/js/main.js [hunk 1]
```diff
@@ -16,6 +16,7 @@ var vgaButton = document.querySelector('#vga');
 var qvgaButton = document.querySelector('#qvga');
 var hdButton = document.querySelector('#hd');
 var fullHdButton = document.querySelector('#full-hd');
+var 4kButton = document.querySelector('#4k');
```
Comment (KaptenJansson) [inline src/content/getusermedia/resolution/js/main.js]: Variables cannot start with numbers.

**Review by KaptenJansson — CHANGES_REQUESTED** [review]

**Comment by KaptenJansson** [comment]
Also I merged the branch with master, you might have to do a git pull before making changes.

File: src/content/getusermedia/resolution/js/main.js (hunk 1 above)
Comment (KaptenJansson) [reply src/content/getusermedia/resolution/js/main.js]: And that's why it red in the github editor as well.

**Comment by rantonysamy** [comment]
Yes, since you merged it with the master, I took that commit "fe45a6eebbbe2bf79f7a" as a base commit and reset my git branch with that, and sent a fixup.

File: src/content/getusermedia/resolution/js/test.js [hunk 2]
```diff
@@ -67,3 +67,7 @@ test('HD capture', function(t) {
 test('FULL HD capture', function(t) {
   resolutionTest(t, 'full-hd', 1920);
 });
+
+test('Four K capture', function(t) {
```
Comment (KaptenJansson) [inline src/content/getusermedia/resolution/js/test.js]: Here you can have "4K" since it's in a string.

Comment (KaptenJansson) [reply src/content/getusermedia/resolution/js/test.js]: But I think either 4K is not enabled on Chrome or the fake camera capture device does not support 4K hence the test times out. I think you can comment out this test and add a comment saying that the fake camera capture device does not support 4K.

You could try this out locally and see if 4K works with a 4K camera at all by modifying the values for the Full HD button on https://webrtc.github.io/samples/src/content/getusermedia/resolution/ in the browser before you click the full HD button. If that works, then try the same using the fake camera ( start Chrome with the command line flag: --use-fake-device-for-media-stream).

**Review by KaptenJansson — APPROVED** [review]

**Pull request merged**